### PR TITLE
Core devs doc and add @emdupre

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
* Document better who the core devs are
* Add @emdupre as a core dev

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -3,38 +3,36 @@
...
+
+.. _core_devs:
+
+Core developers
+.................
+
+The nilearn core developers are:
+
+* Alexandre Gramfort https://github.com/agramfort
+* Ben Cipollini https://github.com/bcipolli
```
Comment (bcipolli) [inline AUTHORS.rst]: See, you do this during Hacktoberfest, and now I feel like it's a good opportunity to make some new contributions :) Will check in soon!

Comment (GaelVaroquaux) [reply AUTHORS.rst]: This was mere luck.

But we are hoping to release soonish. The new HTML reports are totally worth a release!

And we'd be so happy to welcome more of your contributions!

File: AUTHORS.rst [hunk 2]
```diff
@@ -76,4 +74,4 @@ See the scikit-learn documentation on `how to cite
 
 .. |digicomse logo| image:: logos/digi-saclay-logo-small.png
     :height: 25
-    :alt: DigiComse Logo
\ No newline at end of file
+    :alt: DigiComse Logo
```
Comment (aabadie) [inline AUTHORS.rst]: s/DigiComse/DigiCosme/, right ?

Comment (GaelVaroquaux) [reply AUTHORS.rst]: Thanks. Fixed.

File: AUTHORS.rst [hunk 3]
```diff
@@ -45,7 +43,7 @@ Mehdi Rahim, Philippe Gervais where payed by the `NiConnect
 project, funded by the French `Investissement d'Avenir
 <http://www.gouvernement.fr/investissements-d-avenir-cgi>`_.
 
-NiLearn is also supported by `DigiCosme <https://digicosme.lri.fr>`_ |digicomse logo|
+NiLearn is also supported by `DigiCosme <https://digicosme.lri.fr>`_ |digicosme logo|
```
Comment (bthirion) [inline AUTHORS.rst]: +Dataia, since they supported a workshop this year :-)

Comment (GaelVaroquaux) [reply AUTHORS.rst]: Addressed. This should now be ready for merge.

**Comment by bthirion** [comment]
+1 for merge

**Pull request merged**